roles/wp-cli/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 phar_url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
 bin_path: /usr/bin/wp
+wp_cli_completion_url: https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash
```

Comment:
I think this should wait for #392 if we do it since there could be a disconnect between the completion commands.